uiowa/uiowa
# Grad College content type updates

# Updates
## Grad College split
* Added Thesis Defense content type.
* Added the smart date module to handle date field for Thesis Defense.
* Added logic to hide end date and duration fields via `'#access' => FALSE` and logic to correctly set the end date in a custom form submit function.
* Added grad department field to Person content type and scoped it to show on Mentor person type.

# Default platform
* Resolves #2757: Added scoping for `.field_item` `font-size` adjustment to limit it to `div` and `p` tags.

# How to test
* `blt ds --site=grad.uiowa.edu`
* Check to see that Thesis Defense content type exists.
* Create a Thesis Defense item and view it.
* Observe that the Title field displays at the expected size for an `<h2>`.
* Visit the node/add/person page and select Mentor for the person type and observe that the Department field shows up.
